### DISCUSSION TITLE
Свойства организации необходимые лаб группе

### DISCUSSION THREAD

**Discussion by ai-fhir** [fhir-ru/core#75] [Административная РГ]
Нам необходимо, чтобы поля отмеченные на диаграмме были отражены в профилях мед организации

![lab_org](https://user-images.githubusercontent.com/93438901/152654421-f8ee6344-3291-46fb-8efd-89da92f37eaa.jpg)

cc @avzviagin

**Comment by eugenekogan-spb** [comment]
Давайте уточним, важно ли вам знать только юридическое лицо или в каких-то случаях также и подразделение.
Например, для высказывания "направлен Петровым П.П., участковым врачом детской поликлиники №40 Санкт-Петербурга" есть две организации - юридическое лицо ГП№56 и ее структурное подразделение  детская поликлиника №49.

**Reply by avzviagin** [reply]
Из "Руководства по реализации CDA (Release 2) уровень 3. Протокол лабораторного исследования. Редакция 4":
. ОБЯЗАН иметь [1..1] один элемент providerOrganization, который:
I.	ОБЯЗАН иметь один [1..1] элемент id[1] (атрибут @root должен быть представлен с не пустым значением (по справочнику "Реестр медицинских организаций Российской Федерации" (OID: 1.2.643.5.1.13.13.11.1461), поле "OID"), элемент может иметь атрибут @extension, который должен быть представлен с непустым значением, если имеется информация о структурном подразделении медицинской организации (по справочнику "ФРМО. Справочник структурных подразделений" (OID: 1.2.643.5.1.13.13.99.2.114), поле "OID структурного подразделения")), содержащий уникальный идентификатор медицинской организации и уникальный идентификатор структурного подразделения;

**Reply by eugenekogan-spb** [reply]
Спасибо, ясно.

**Comment by eugenekogan-spb** [comment]
И еще вопрос: в сведениях о лицензии достаточно ли указать ее номер, сроки действия, выдавшую организацию?

**Reply by avzviagin** [reply]
Из "Руководства по реализации CDA (Release 2) уровень 3. Протокол лабораторного исследования. Редакция 4":
II.	МОЖЕТ содержать один [0..1] элемент id[2], который соответствует лицензии на осуществление медицинской деятельности (атрибут @root должен иметь значение "1.2.643.5.1.13.2.1.1.1504.101", атрибут @extension должен содержать номер лицензии, @assigningAuthorityName должен содержать наименование организации, выдавшей лицензию, и дату выдачи лицензии);

**Reply by eugenekogan-spb** [reply]
Спасибо, ясно. Значит, подойдет Identifier.

**Comment by ai-fhir** [comment]
Дополнительно прикладываю примеры CDA, чтобы можно было посмотреть используемые там значения

[Протокол Лаб Исследования CDA](https://github.com/fhir-ru/pli/blob/main/assets/CDA_%D0%9F%D0%A0%D0%9E%D0%A2%D0%9E%D0%9A%D0%9E%D0%9B_%D0%9B%D0%90%D0%91%D0%9E%D0%A0%D0%90%D0%A2%D0%9E%D0%A0%D0%9D%D0%9E%D0%93%D0%9E_%D0%98%D0%A1%D0%A1%D0%9B%D0%95%D0%94%D0%9E%D0%92%D0%90%D0%9D%D0%98%D0%AF_%D0%A04.docx)

[Протокол лабораторного исследования пример максимальный xml](https://github.com/fhir-ru/pli/blob/main/assets/%D0%9F%D1%80%D0%BE%D1%82%D0%BE%D0%BA%D0%BE%D0%BB_%D0%BB%D0%B0%D0%B1%D0%BE%D1%80%D0%B0%D1%82%D0%BE%D1%80%D0%BD%D0%BE%D0%B3%D0%BE_%D0%B8%D1%81%D1%81%D0%BB%D0%B5%D0%B4%D0%BE%D0%B2%D0%B0%D0%BD%D0%B8%D1%8F_%D0%BF%D1%80%D0%B8%D0%BC%D0%B5%D1%80_%D0%BC%D0%B0%D0%BA%D1%81%D0%B8%D0%BC%D0%B0%D0%BB%D1%8C%D0%BD%D1%8B%D0%B9.xml)
